components/Rating/rating.js

```diff
@@ -0,0 +1,14 @@
+const ratingStars = document.getElementsByClassName("rating-star");
+for (let i = 0;i < ratingStars.length;i++){
+    ratingStars[i].addEventListener("click",(e) => {
+        const id = ratingStars[i].id;
+        const idValue = id.slice(id.length-1);
+        console.log(idValue)
```

Comment:
Just a suggestion You can remove the console log here

Reply:
will fix that!